staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/BUILD

```diff
@@ -19,6 +19,7 @@ go_library(
         "//staging/src/k8s.io/client-go/transport:go_default_library",
         "//staging/src/k8s.io/client-go/util/connrotation:go_default_library",
         "//vendor/github.com/golang/glog:go_default_library",
+        "//vendor/github.com/prometheus/client_golang/prometheus:go_default_library",
```

Comment:
does this add prometheus to the dependencies of every client-go user? Or did we have that before already?

Reply:
client-go has not imported prometheus before.
so anything that ends up importing client-go/transport or client-go/rest will get this added to its dependencies.

Reply:
that's arguably a pretty big vendor path.